pkg/kubelet/kubelet_node_status.go

```diff
@@ -644,7 +644,6 @@ func (kl *Kubelet) patchNodeStatus(originalNode, node *v1.Node) (*v1.Node, error
 		return nil, err
 	}
 	kl.lastStatusReportTime = kl.clock.Now()
-	kl.setLastObservedNodeAddresses(updatedNode.Status.Addresses)
```

Comment:
@cartermckinnon this will also improve the latency on cloud providers, since it will not have to wait for the patch of the kubelet,